packages/Tabs/src/Tabs.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import PropTypes from "prop-types";
+import TabsContext from "./TabsContext";
+import Panel from "./components/Panel/Panel";
+import Panels from "./components/Panels/Panels";
+import Tab from "./components/Tab/Tab";
+import List from "./components/List/List";
+
+const propTypes = {
+  children: PropTypes.node.isRequired,
+  isDisabled: PropTypes.node,
+};
+
+const defaultProps = {
+  isDisabled: false,
+};
+
+const Tabs = props => {
+  const [activeIndex, setActiveIndex] = React.useState(null);
```

Comment:
Hmm. It looks like it's impossible for me to render tabs with a specific tab pre-selected? It always starts out with no tab selected because it's uncontrolled and this default value is `null`.